test/entry-departure.test.ts

```diff
@@ -19,4 +20,31 @@ describe('DepartureEntry', () => {
     expect(validationErrors).toBeDefined();
     expect(validationErrors.length).toBeGreaterThan(0);
   });
+
+  test('Validating an minimal DepartureEntry should succeed', () => {
```

Comment:
Sorry for grammar issue but it's 'a minimal...'

Reply:
Fixed